src/components/CompareResults/SubtestsResults/SubtestsResultsMain.tsx

```diff
@@ -145,25 +134,75 @@ function SubtestsResultsMain({ view }: SubtestsResultsMainProps) {
           </Link>
           {'.'}
         </Alert>
-        <SubtestsRevisionHeader header={subtestsHeader} view={view} />
-        <Grid container spacing={1}>
-          <Grid item xs={12} md={6} sx={{ marginInlineEnd: 'auto' }}>
-            <SearchInput
-              defaultValue={initialSearchTerm}
-              onChange={onSearchTermChange}
-            />
-          </Grid>
-          <Grid item xs='auto'>
-            <DownloadButton resultsPromise={[results]} />
-          </Grid>
-          <Grid item xs='auto'>
-            <RetriggerButton result={results[0]} variant='text' />
-          </Grid>
-        </Grid>
+
+        <Suspense
+          fallback={
+            <>
+              <Stack spacing={1} sx={{ marginBottom: '12px' }}>
+                <Skeleton
+                  variant='rounded'
+                  sx={{
+                    // This fontSize value makes the skeleton match the height of the replaced element.
+                    fontSize: '2.18em',
+                    backgroundColor: secondary.main,
```

Comment:
just use "secondary.main" as a string, then material UI knows how to do it inrternally :-)